docs/src/main/asciidoc/http-reference.adoc

```diff
@@ -69,9 +69,11 @@ The response body of a static resource is not compressed by default.
 You can enable the HTTP compression support by means of `quarkus.http.enable-compression=true`.
 If compression support is enabled then the response body is compressed if the `Content-Type` header derived from the file name of a resource is a compressed media type as configured via `quarkus.http.compress-media-types`.
 
-TIP: By default, the following list of media types is compressed: `text/html`, `text/plain`, `text/xml`, `text/css`, `text/javascript` and `application/javascript`.
+TIP: By default, the following list of media types is compressed: `text/html,text/plain,text/xml,text/css,text/javascript,application/javascript,application/graphql+json`. It means some other noteworthy MIME types such as `application/json,application/xhtml+xml` are NOT compressed by default.
```

Comment:
I'm not convinced having a unique list with commas is more readable. I would prefer we keep the list as it was.

Should we consider adding the other two you mentioned to the default media types we compress? I'm not asking that for this PR but would it make sense to open an issue as a follow-up? (open question, I haven't looked at the details).

Also `s/MIME/media/` for consistency.